events/static/events/create_event.css

```diff
@@ -0,0 +1,245 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500;700&display=swap");
+
+* {
+  transition: 0.1s ease;
+}
+
+body {
+  background: url("https://images.unsplash.com/photo-1480497490787-505ec076689f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1738&q=80") no-repeat;
+  background-size: cover;
+  font-family: "Montserrat", sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container-shadow {
+  position: absolute;
+  width: 600px;
+  height: 900px;
+  left: 48%;
+  margin-left: -262.5px;
+  margin-top: 10%;
+  box-shadow: 0px 30px 50px -20px #000;
+}
+
+.container {
+  position: absolute;
+  width: 600px;
+  height: 900px;
+  left: 48%;
+  margin-left: -262.5px;
+  margin-top: 10%;
+  background: linear-gradient(90deg, #29323c 0%, #3b4249 100%);
+  box-shadow: 0px 0px 50px -20px #000;
+}
+
+.wrap {
+  position: relative;
+  width: 400px;
+  margin: 0 auto;
+}
+
+.headings {
+  position: relative;
+  margin: 90px auto 70px;
+}
+
+.headings > a {
+  color: rgba(255, 255, 255, 0.7);
+  padding-bottom: 10px;
+  border-bottom: solid 2px rgba(17, 97, 237, 0);
+  display: inline-block;
+}
+
+.headings > a:first-child {
+  margin-left: 20px;
+  margin-right: 45px;
+}
+
+.headings > a:hover {
+  color: rgba(255, 255, 255, 1);
+}
+
+.headings > a.active {
+  color: rgba(255, 255, 255, 1);
+  border-bottom: solid 4px #aae0f0;
+  box-shadow: 0 5px 5px -5px #1d2124;
+}
+
+.headings > a span {
+  font-size: 1.4rem;
+  letter-spacing: 1px;
+}
+
+label {
+  color: rgba(255, 255, 255, 0.8);
+  text-transform: uppercase;
+  padding: 13px 20px;
+  position: relative;
+  display: block;
+  font-size: 14px;
+}
+
+input {
+  width: 100%;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  letter-spacing: 1px;
+  background: transparent;
+  padding: 10px 20px;
+  border: none;
+  border-bottom: 2px solid #1d2124;
+  outline: none;
+  box-sizing: border-box;
+
+  margin-bottom: 10px;
+}
+
+input:active,
+input:focus {
+  border-bottom: 0.12em solid #aae0f0;
+  border-color: #aae0f0;
+  box-shadow: 0 5px 5px -5px #aae0f0;
+}
+
+#title {
+  margin-top: -10px;
+}
+
+#input-title {
+  font-size: 1.3em;
+}
+
+#location {
+  font-size: 1.3em;
+}
+
+#id_color {
+  margin-left: 20px;
+  font-size: 1.3em;
+}
+
+#description {
+  width: 380px;
+  margin-left: 10px;
+  font-size: 1.3em;
+}
+
+#password + i {
+  color: rgba(255, 255, 255, 0.7);
+  font-size: 25px;
+  float: right;
+  position: relative;
+  top: -43px;
+  right: 15px;
+  cursor: pointer;
+}
+
+#password + i:hover {
+  color: rgba(255, 255, 255, 1);
+}
```

Comment:
In this Form there is no Password id,
Can you remove these lines if you don't use them in the HTML.

Reply:
done